Jenkinsfile

```diff
@@ -14,9 +14,18 @@ def bazel = new Bazel()
 // This should be updated for a release branch.
 ISTIO_VERSION_URL = 'https://raw.githubusercontent.com/istio/istio/master/istio.RELEASE'
 
+
+def setVersions() {
+  def version = sh(returnStdout: true, script: "curl ${ISTIO_VERSION_URL}").trim()
+  def v = version.tokenize('.')
+  env.ISTIO_VERSION = version
+  env.ISTIO_MINOR_VERSION = "${v[0]}.${v[1]}"
```

Comment:
This is the latest non-patch version of the current release cycle, correct? For example, given x.y.z, ISTIO_MINOR_VERSION would always be x.y?